server/base/src/main/java/org/apache/accumulo/server/manager/state/TabletManagementScanner.java

```diff
@@ -41,6 +47,7 @@
 public class TabletManagementScanner implements ClosableIterator<TabletManagement> {
 
   private static final Logger log = LoggerFactory.getLogger(TabletManagementScanner.class);
+  private static final Collection<Range> ALL_TABLETS_RANGE = List.of(new Range());
```

Comment:
Might be able to use MetadataSchema.TabletsSection.getRange() to limit this to the tablet metadata range.

Reply:
Implement in 2b83d93